examples/watcher_kind/README.me

```diff
@@ -0,0 +1,3 @@
+# Watcher kind example
```

Comment:
Typo on file extension?

Reply:
uh yeah, also thanks for reviewing but just to make sure: It's not finished, just had some time yesterday night

Reply:
Alright, ping me if it's ready :)